src/main/java/it/gov/pagopa/pu/migration/wf/dto/debtposition/InstallmentIngestionFlowFileDTO.java

```diff
@@ -0,0 +1,124 @@
+import com.opencsv.bean.CsvBindByName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class InstallmentIngestionFlowFileDTO {
+
+    @CsvBindByName(column = "IUPD")
+    private String iupdOrg;
+
+    @CsvBindByName(column = "descrizionePosizioneDebitoria")
+    private String description;
+
+    @CsvBindByName(column = "dataValidita")
+    private String validityDate;
+
+    @CsvBindByName(column = "multiDebtor")
+    private String multiDebtor;
+
+    @CsvBindByName(column = "dataNotifica")
+    private String notificationDate;
+
+    @CsvBindByName(column = "indiceOpzionePagamento")
+    private String paymentOptionIndex;
+
+    @CsvBindByName(column = "tipoOpzionePagamento")
+    private String paymentOptionType;
+
+    @CsvBindByName(column = "descrizioneOpzionePagamento")
+    private String paymentOptionDescription;
+
+    @CsvBindByName(column = "iud")
+    private String iud;
+
+    @CsvBindByName(column = "codIUV")
+    private String iuv;
+
+    @CsvBindByName(column = "tipoIdentificativoUnivoco")
+    private String entityType;
+
+    @CsvBindByName(column = "codiceIdentificativoUnivoco")
+    private String fiscalCode;
+
+    @CsvBindByName(column = "anagraficaPagatore")
+    private String fullName;
+
+    @CsvBindByName(column = "indirizzoPagatore")
+    private String address;
+
+    @CsvBindByName(column = "civicoPagatore")
+    private String civic;
+
+    @CsvBindByName(column = "capPagatore")
+    private String postalCode;
+
+    @CsvBindByName(column = "localitaPagatore")
+    private String location;
+
+    @CsvBindByName(column = "provinciaPagatore")
+    private String province;
+
+    @CsvBindByName(column = "nazionePagatore")
+    private String nation;
+
+    @CsvBindByName(column = "emailPagatore")
+    private String email;
+
+    @CsvBindByName(column = "dataEsecuzionePagamento")
+    private String dueDate;
+
+    @CsvBindByName(column = "importoDovuto")
+    private String amount;
+
+    @CsvBindByName(column = "tipoDovuto")
+    private String debtPositionTypeCode;
+
+
+    @CsvBindByName(column = "causaleVersamento")
+    private String remittanceInformation;
+
+    @CsvBindByName(column = "datiSpecificiRiscossione")
+    private String legacyPaymentMetadata;
+
+    @CsvBindByName(column = "flagGeneraIuv")
+    private String flagPuPagoPaPayment;
+
+    @CsvBindByName(column = "bilancio")
+    private String balance;
+
+    @CsvBindByName(column = "flagMultiBeneficiario")
+    private String flagMultiBeneficiary;
+
+    @CsvBindByName(column = "numeroBeneficiari")
+    private String numberBeneficiary;
+
+    @CsvBindByName(column = ".*_2")
+    private String transfer2;
+
+    @CsvBindByName(column = ".*_3")
+    private String transfer3;
+
+    @CsvBindByName(column = ".*_4")
+    private String transfer4;
+
+    @CsvBindByName(column = ".*_5")
+    private String transfer5;
```

Comment:
I think you cannot do this... probably you should define all the columns supposed to be intercepted by this regexp